.github/workflows/manifold.yml

```diff
@@ -78,6 +78,7 @@ jobs:
         files: build/code_coverage.info
         fail_ci_if_error: false
         name: ${{matrix.parallelization}}
+        token: ${{secrets.CODECOV_TOKEN}}
```

Comment:
```suggestion
        token: ${{secrets.CODECOV_TOKEN}}
        verbose: true
```

Reply:
Done - output should be ready in 10 min or so.

Reply:
From https://github.com/elalish/manifold/actions/runs/11805031296/job/32886589779?pr=1032
```
gpg: directory '/home/runner/.gnupg' created
gpg: keybox '/home/runner/.gnupg/pubring.kbx' created
gpg: /home/runner/.gnupg/trustdb.gpg: trustdb created
gpg: key 806BB28AED779869: public key "Codecov Uploader (Codecov Uploader Verification Key) <security@codecov.io>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: Signature made Fri Oct 18 16:02:53 2024 UTC
gpg:                using RSA key 27034E7FDB850E0BBC2C62FF806BB28AED779869
gpg: Good signature from "Codecov Uploader (Codecov Uploader Verification Key) <security@codecov.io>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 2703 4E7F DB85 0E0B BC2C  62FF 806B B28A ED77 9869
==> Uploader SHASUM verified (452ad3c57dc4ff698e9ca36786a2d09be5db254c32c4a4fb6d08c18146e4d8c1  codecov)
==> Running version latest
==> Running version v0.8.0
==> Running git config --global --add safe.directory /home/runner/work/manifold/manifold
/usr/bin/git config --global --add safe.directory /home/runner/work/manifold/manifold
==> Running command '/home/runner/work/_actions/codecov/codecov-action/v4/dist/codecov -v create-commit'
/home/runner/work/_actions/codecov/codecov-action/v4/dist/codecov -v create-commit --git-service github -C f7f7222e03f972ec0c708143f43[40](https://github.com/elalish/manifold/actions/runs/11805031296/job/32886589779?pr=1032#step:10:41)5544f4158fe
info - 2024-11-12 20:29:03,575 -- ci service found: github-actions
debug - 2024-11-12 20:29:03,577 -- versioning system found: <class 'codecov_cli.helpers.versioning_systems.GitVersioningSystem'>
debug - 2024-11-12 20:29:03,579 -- versioning system found: <class 'codecov_cli.helpers.versioning_systems.GitVersioningSystem'>
debug - 2024-11-12 20:29:03,581 -- Loading config from /home/runner/work/manifold/manifold/.codecov.yml
debug - 2024-11-12 20:29:03,582 -- Starting create commit process --- {"verbose": true, "auto_load_params_from": null, "codecov_yml_path": null, "enterprise_url": null, "version": "cli-0.8.0", "command": "create-commit", "git_service": "github", "commit_sha": "f7f7222e03f972ec0c708143f43405544f[41](https://github.com/elalish/manifold/actions/runs/11805031296/job/32886589779?pr=1032#step:10:42)58fe", "parent_sha": null, "pull_request_number": "1032", "branch": "codecov", "fail_on_error": false, "slug": "elalish/manifold"}
info - 2024-11-12 20:29:04,251 -- Process Commit creating complete
debug - 2024-11-12 20:29:04,252 -- Commit creating result --- {"result": "RequestResult(error=RequestError(code='HTTP Error 400', params={}, description='{\"message\":\"Repository not found\"}\\n'), warnings=[], status_code=400, text='{\"message\":\"Repository not found\"}\\n')"}
error - 2024-11-12 20:29:04,252 -- Commit creating failed: {"message":"Repository not found"}
```

Reply:
I see two lines that look fishy to me because their SHAs don't match:
```
f7f7222e03f972ec0c708143f43[40](https://github.com/elalish/manifold/actions/runs/11805031296/job/32886589779?pr=1032#step:10:41)5544f4158fe
```
vs
```
f7f7222e03f972ec0c708143f43405544f[41](https://github.com/elalish/manifold/actions/runs/11805031296/job/32886589779?pr=1032#step:10:42)58fe
```

Reply:
just to update, tackling with another team member to figure out what's going on